webcompat/views.py

```diff
@@ -350,7 +350,7 @@ def contributors():
 
 @app.route('/contributors/report-bug')
 @cache_policy(private=True, uri_max_age=0, must_revalidate=True)
-def contributors_report_bug():
+def contributors_bug_report():
```

Comment:
@miketaylr Probably these changes in `views.py` are not necessary. See my comment on the templates.